x-pack/plugins/security_solution/public/common/components/event_details/event_details.tsx

```diff
@@ -73,6 +112,17 @@ export const EventDetails = React.memo<Props>(
 
     return (
       <Details data-test-subj="eventDetails">
+        <PopoverContainer>
+          <EuiPopover
+            button={button}
+            isOpen={false}
+            closePopover={noop}
+            style={popoverStyle}
+            repositionOnScroll={true}
+          >
+            {null}
```

Comment:
it is strange to use EuiPopover with nothing inside. Can we use a EuiButton component?

Reply:
It agree it looks strange, but the button must be passed to `EuiPopover` via the `button` prop, not it's children. This usage of `EuiPopover` is based on the [Popover on a fixed element example](https://elastic.github.io/eui/#/layout/popover).

Reply:
Per our out-of-band conversation, we agreed the following fix combined with the previous comment is better:

```
        <PopoverContainer>
          <EuiPopover
            button={button}
            isOpen={false}
            closePopover={noop}
            repositionOnScroll={true}
          />
        </PopoverContainer>
```